tests/models/mllama/test_modeling_mllama.py

```diff
@@ -537,9 +538,18 @@ def test_11b_model_integration_generate(self):
 
         inputs = processor(text=prompt, images=image, return_tensors="pt").to(torch_device)
 
+        input_ids = inputs["input_ids"]
+
         # Check inputs ids
-        expected_input_ids = torch.tensor([[128256, 128000, 2746, 358, 1047, 311, 3350, 264, 6520, 39342, 369, 420, 832]], device=torch_device)  # fmt: skip
-        self.assertTrue(torch.equal(inputs["input_ids"], expected_input_ids))
+        expected_input_ids_all = Expectations(
+            {
+                ("xpu", 3): torch.tensor([[128000, 128256, 128000, 2746, 358, 1047, 311, 3350, 264, 6520, 39342, 369, 420, 832]], device=torch_device),
+                ("cuda", 7): torch.tensor([[128256, 128000, 2746, 358, 1047, 311, 3350, 264, 6520, 39342, 369, 420, 832]], device=torch_device),
```

Comment:
I don't think tokenizer output will be different across devices, and i guess T4 expectations need to be updated(it's SLOW cases, so maybe need you verify on T4, since I don't have T4 access). For backward-compatibility, I keep original expectations here.

Reply:
yes, thank you.